tokio/src/sync/oneshot.rs

```diff
@@ -331,6 +331,7 @@ pub mod error {
     use std::fmt;
 
     /// Error returned by the `Future` implementation for `Receiver`.
+    /// This error is returned when the sender is dropped without sending.
```

Comment:
You have to put an empty line for there to be a newline in the generated documentation.
```suggestion
    /// Error returned by the `Future` implementation for `Receiver`.
    ///
    /// This error is returned by the receiver when the sender is dropped without sending.
```